README.md

```diff
@@ -1,8 +1,34 @@
 
 # penalties-file-notification-orchestrator
 
-This is a placeholder README.md for a new repository
+PENALTIES FILE NOTIFICATION ORCHESTRATOR has 2 endpoints
 
-### License
+POST        /new-notifications                    controllers.OrchestratorController.receiveSDESNotifications
+new-notifications is a new notification service for backend services.
+
+
+POST        /sdes-callback                        controllers.SDESCallbackController.handleCallback
+sde will pull from file upload, process the file and message is sent back with the status.
+
+
+## Running
+
+This application runs on port 9184.
+
+You can use the ./run.sh to run the service.
+
+The user must have an authenticated session and be enrolled in MTD VAT to access most pages of this service.
+
+The service manager configuration name for this service is: PENALTIES_FILE_NOTIFICATION_ORCHESTRATOR
+
+This service is dependent on other services, all dependent services can be started with sm --start PENALTIES_ALL (this will also start the penalties frontend microservice so you may need to stop it via sm --stop PENALTIES_FILE_NOTIFICATION_ORCHESTRATOR).
```

Comment:
Slight wording change needed:
```suggestion
This service is dependent on other services, all dependent services can be started with sm --start PENALTIES_ALL (this will also start the penalties file notification orchestrator microservice so you may need to stop it via sm --stop PENALTIES_FILE_NOTIFICATION_ORCHESTRATOR).
```
```suggestion
This service is dependent on other services, all dependent services can be started with sm --start PENALTIES_ALL (this will also start the penalties frontend microservice so you may need to stop it via sm --stop PENALTIES_FILE_NOTIFICATION_ORCHESTRATOR).
```